quizcomp/quiz.py

```diff
@@ -206,3 +206,15 @@ def create_variant(self, identifier = None, seed = None, all_questions = False):
         data['_skip_class_validations'] = [Quiz]
 
         return quizcomp.variant.Variant(**data)
+    
+    @property
```

Comment:
We generally don't use `@property` in this repo, since it can hide computational complexity.